scripts/anvil_tools/batch_upsert_entities_standard.py

```diff
@@ -72,66 +96,58 @@ def create_upsert_request(tsv, array_attr_cols=None):
     # this is specific just to the first column where the format is required for terra load tsv files
     tsv.rename(columns={entity_type_col_name: entity_type}, inplace=True)
 
-    # templates
-    # for request body skeleton
-    template_req_body = """{"name":"VAR_ENTITY_ID", "entityType":"VAR_ENTITY_TYPE", "operations":[OPERATIONS_LIST]}"""
-
-    # for operations - create/update: list attribute, add list member to attribute if list, single attribute and value
-    template_make_list_attr = '{"op":"CreateAttributeValueList","attributeName":"VAR_ATTRIBUTE_LIST_NAME"},'
-    template_add_list_member = '{"op":"AddListMember","attributeListName":"VAR_ATTRIBUTE_LIST_NAME", "newMember":"VAR_LIST_MEMBER"},'
-    template_make_single_attr = '{"op":"AddUpdateAttribute","attributeName":"VAR_ATTRIBUTE_NAME", "addUpdateAttribute":"VAR_ATTRIBUTE_MEMBER"},'
-
-    # initiate string to capture all operation requests
-    all_attributes_request = []
+    # initiate string to capture all operation requests for all rows (entities) in given tsv file
+    all_entities_request = []
 
+    # for every row (entity) in df
     for index, row in tsv.iterrows():
-        single_attribute_request = ''''''
+        # initialize string to capture request for one row (entity) in tsv
+        single_entity_operations = ''''''
         # get the entity_id (row id - must be unique)
         entity_id = str(row[0])
 
-        # if array columns/attributes
+        # if array columns/attributes provided
         if array_attr_cols:
             # for each array column
             for col in array_attr_cols:
-                # add column name to template for a single attribute
-                single_attribute_request += template_make_list_attr.replace("VAR_ATTRIBUTE_LIST_NAME", col)
+                # get operation json to make the array/list attribute with column name
+                single_entity_operations += create_list_attr_operation(col)
```

Comment:
as per other comment, remove comma from functions and add it here, i.e. `single_entity_operations += create_list_attr_operation(col) + ","`